roles/download/defaults/main.yml

```diff
@@ -167,6 +167,8 @@ contiv_ovs_image_repo: "contiv/ovs"
 contiv_ovs_image_tag: "latest"
 cilium_image_repo: "docker.io/cilium/cilium"
 cilium_image_tag: "{{ cilium_version }}"
+cilium_init_image_repo: "docker.io/library/busybox"
```

Comment:
You need to add this in the `download` dict too. So it actually get's downloaded by the role.